CHANGELOG.md

```diff
@@ -11,10 +11,17 @@ MOVE THIS SECTION TO THE CORRECT LOCATION
 ## Unreleased
 
 ### Bug Fixes
-- Return Swift optionals from Objective C methods that can return nil
+
+- **AWSMobileClient**
+  - fix(AWSMobileClient): eliminates thread blocking in operations ([PR #3872](https://github.com/aws-amplify/aws-sdk-ios/pull/3872))
 
 ## 2.26.5
 
+- **AWSCore**
+  - Support for `ap-southeast-3` - Asia Pacific (Jakarta) (see [AWS Regional Services List](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/) for a list of services supported in the region)
+
+- Return Swift optionals from Objective C methods that can return nil
```

Comment:
```suggestion
- Return Swift optionals from Objective C methods that can return nil ([PR #3912](https://github.com/aws-amplify/aws-sdk-ios/pull/3912))

```

Reply:
I've rebuilt the branch and used fixup to make it a single commit without updates to CHANGELOG.md to avoid the merge conflict. now it is just the code changes and the Xcode project file.